tests/test_permissions_cli.py

```diff
@@ -0,0 +1,500 @@
+import json
+
+import responses
+
+
+# GET
+@responses.activate
+def test_permissions_get(cli_run):
+    responses.add(
+        responses.GET,
+        "https://analysispreservation-dev.cern.ch/api/deposits/some-pid",
+        json={
+            'permissions': {
+                'deposit-admin': {
+                    'roles': [1],
+                    'users': ['cms@inveniosoftware.org']
+                },
+                'deposit-read': {
+                    'roles': [],
+                    'users': ['cms@inveniosoftware.org']
+                },
+                'deposit-update': {
+                    'roles': [],
+                    'users': ['cms@inveniosoftware.org']
+                }
+            }
+        },
+        status=200)
+
+    res = cli_run("permissions get -p some-pid")
+
+    assert responses.calls[0].request.headers[
+        'Accept'] == 'application/permissions+json'
+    assert res.exit_code == 0
+    assert json.loads(res.stripped_output) == {
+        'deposit-admin': {
+            'roles': [1],
```

Comment:
i think that's not how it looks like anymore, check [here](https://github.com/cernanalysispreservation/analysispreservation.cern.ch/pull/1613)